lib/recipe/view/edit_probe_page.dart

```diff
@@ -12,29 +12,32 @@ class EditProbePage extends StatelessWidget {
   Widget build(BuildContext context) {
     return BlocBuilder<RecipeBloc, RecipeState>(builder: (context, state) {
       return Scaffold(
-          appBar: AppBar(title: const Text('Edit Probe')),
-          floatingActionButton: Column(
-              mainAxisAlignment: MainAxisAlignment.end,
-              crossAxisAlignment: CrossAxisAlignment.end,
-              children: <Widget>[
-                FloatingActionButton.extended(
-                    heroTag: null,
-                    label: const Text("Add probe response"),
-                    icon: const Icon(Icons.add),
-                    onPressed: () => {
-                          context.read<RecipeBloc>().add(ProbeOptionAdded(
-                              recipe: state.recipes[recipeIndex],
-                              probeIndex: probeIndex)),
-                        }),
-                const SizedBox(
-                  height: 10,
-                ),
-                FloatingActionButton.extended(
-                    heroTag: null,
-                    label: const Text("Save Probe"),
-                    icon: const Icon(Icons.save_sharp),
-                    onPressed: () => {Navigator.pop(context)})
-              ]),
+          appBar: AppBar(title: const Text('View Probe')),
```

Comment:
Shouldn't the AppBar title by dynamic depending on whether we are editing or viewing the probe? Either way, we might want to rename the file to reflect that, or just have it as `probe_page.dart` just as there is `ingredient_page.dart`.

Reply:
Yeah I did consider this, but imo there's not much of a difference between having the different titles. Sometimes you're clicking into the page to view it, sometimes you're clicking into it to edit it. I think having the generic 'View' and the ability to edit being implicit is alright personally

Reply:
Yup, sounds good. So then we should change the filename to `probe_page.dart`.